lib/logger/src/styles.ts

```diff
@@ -16,9 +21,22 @@ export const styles = {
   dim: colours.grey,
   title: colours.bold.underline,
   taskHeader: colours.bgWhite.black,
-  errorHighlight: colours.red,
-  error: (string: string): string => `${styles.errorHighlight.bold('‼︎')} ${styles.title(string)}`,
-  warningHighlight: colours.yellow,
-  warning: (string: string): string => styles.warningHighlight.bold('⚠︎') + ' ' + string,
-  ruler: (): string => styles.dim('─'.repeat(process.stdout.columns / 2))
+  errorHighlight: colours.bgRed.bold,
+  error: (string: string): string => `${styles.errorHighlight(' × ️')} ${styles.title(string)}`,
+  warningHighlight: colours.bgYellow.bold,
+  warning: (string: string): string => styles.warningHighlight(' ! ') + ' ' + string,
+  infoHighlight: colours.bgBlueBright.bold,
+  info: (string: string): string => styles.infoHighlight(' i ') + ' ' + string,
+  helpHighlight: colours.bgGreen.bold,
+  help: (string: string): string => styles.helpHighlight(' ? ') + ' ' + string,
+  ruler: (): string => styles.dim('─'.repeat(process.stdout.columns / 2)),
+  box: (string: string, options: Partial<BoxenOptions>) =>
+    boxen(string, {
+      borderStyle: 'round',
+      padding: { left: 1, right: 1 },
+      ...options
+    }),
+  groupHeader: (string: string) => ` ╭─${'─'.repeat(stripAnsi(string).length)}─╮
```

Comment:
**nitpick**: this length calculation logic [won't be able to handle emojis or non-english characters](https://tonsky.me/blog/unicode/), and we should technically be using a library like [`string-width`](https://github.com/sindresorhus/string-width). buuuuuuut we're probably only going to be using ASCII right?

Reply:
ah. let me guess, `string-width` is ESM-only? i'm not sure how bothered i can be about this tbh

Reply:
yep, probably not worth it